packages/core/src/lib/icon/icons.ts

```diff
@@ -116,6 +116,12 @@ export const paths = {
     'M17 22v-2h3v-3h2v3.5c0 .4-.2.7-.5 1-.3.3-.6.5-1 .5H17ZM7 22H3.5c-.4 0-.7-.2-1-.5-.3-.3-.5-.6-.5-1V17h2v3h3v2ZM17 2h3.5c.4 0 .7.2 1 .5.3.3.5.6.5 1V7h-2V4h-3V2ZM7 2v2H4v3H2V3.5c0-.4.2-.7.5-1 .3-.3.6-.5 1-.5H7Zm6 15.3 4-2.4v-4.5l-4 2.3v4.6Zm-1-6.4 4-2.3-4-2.3-4 2.3 4 2.3Zm-5 4 4 2.3v-4.5l-4-2.3v4.5Zm11.2-7.3c.5.3.8.7.8 1.3v6.3c0 .6-.3 1-.8 1.4l-5.4 3.1c-.6.4-1 .4-1.6 0l-5.4-3.1c-.5-.4-.8-.8-.8-1.4V9c0-.6.3-1 .8-1.3l5.5-3.2a1.6 1.6 0 0 1 1.4 0l5.5 3.2Z',
   'viam-local-module':
     'M3 3v18h18V3H3Zm5.6 9.9H5V5h8v3.6H8.6V13Zm-3.6 6h14V5h-4v5.6h-4.4V15H5v4Z',
+  'viam-mixed-part-status': {
+    path1:
+      'M8 7.33341C7.26667 7.33341 6.66667 7.93341 6.66667 8.66675C6.66667 9.40008 7.26667 10.0001 8 10.0001C8.73333 10.0001 9.33333 9.40008 9.33333 8.66675C9.33333 7.93341 8.73333 7.33341 8 7.33341ZM12 8.66675C12 6.46008 10.2067 4.66675 8 4.66675C5.79333 4.66675 4 6.46008 4 8.66675C4 10.1467 4.80667 11.4334 6 12.1267L6.66667 10.9667C5.87333 10.5001 5.33333 9.65341 5.33333 8.66675C5.33333 7.19341 6.52667 6.00008 8 6.00008C9.47333 6.00008 10.6667 7.19341 10.6667 8.66675C10.6667 9.65341 10.1267 10.5001 9.33333 10.9667L10 12.1267C11.1933 11.4334 12 10.1467 12 8.66675Z',
+    path2:
+      'M8.00016 2C4.32016 2 1.3335 4.98667 1.3335 8.66667C1.3335 11.1333 2.6735 13.28 4.66016 14.4333L5.32683 13.28C3.74016 12.3533 2.66683 10.64 2.66683 8.66667C2.66683 5.72 5.0535 3.33333 8.00016 3.33333C10.9468 3.33333 13.3335 5.72 13.3335 8.66667C13.3335 10.64 12.2602 12.3533 10.6668 13.28L11.3335 14.4333C13.3268 13.28 14.6668 11.1333 14.6668 8.66667C14.6668 4.98667 11.6802 2 8.00016 2Z',
+  },
```

Comment:
I like this! Let's get a little more generic, since two-tone icons seem like a thing that we'll want more of! Let's make the data look like:

```ts
'viam-mixed-part-status': [
  {
    path: '...'
  },
  {
    path: '...',
    opacity: '0.5'
  }
]
```

(Some extra type definitions may be necessary, that's ok)

Then, instead of checking for `{#if name === 'viam-mixed-part-status'}`, we can do something like:

```svelte
<script lang="ts">
// ...

$: allPaths = typeof paths[name] === 'string'
  ? [{ path: paths[name] }]
  : paths[name]
</script>

<svg
  class={cx(sizes[size], extraClasses)}
  viewBox="0 0 24 24"
  aria-hidden="true"
  focusable="false"
  {...$$restProps}
>
  {#each allPaths as { path: d, opacity }}
    <path
      {d}
      {opacity}
      fill-rule="evenodd"
      fill="currentColor"
    />
  {/each}
</svg>

Reply:
Thanks!